data/music_albums.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "publish_date": "2022-12-12",
+    "genre": "Rock",
+    "label_title": "Albumt",
+    "label_color": "Red",
+    "author_first_name": "Auth",
+    "author_last_name": "LasAuth",
+    "on_spotify": true
+  },
+  {
+    "publish_date": "2011-11-11",
+    "genre": "roca",
+    "label_title": "Albu1",
+    "label_color": "brown",
+    "author_first_name": "nic",
+    "author_last_name": "ol",
+    "on_spotify": true
+  }
+]
```

Comment:
- Great Job for being able to display the JSON data, I think you forget to save what you added, so please save it to a JSON file to be able to get what we added after quitting the app 👍